src/applications/personalization/dashboard/sass/dashboard-alert.scss

```diff
@@ -57,61 +90,49 @@
     }
   }
 
-  &-closed section.status {
-    border-color: $color-black;
-
-    h3::before {
-      content: fa-content($fa-var-exclamation-circle);
-      color: $color-black;
-    }
-  }
-
-  &-decision section.status {
-    border-color: $color-red;
-
-    h3::before {
-      content: fa-content($fa-var-exclamation-circle);
-      color: $color-red;
-    }
-  }
-
-  &-enrolled section.status {
-    border-color: $color-green;
-
-    h3::before {
-      content: fa-content($fa-var-check-circle);
-      color: $color-green;
-    }
-  }
-
-  &-in-progress section.status {
-    border-color: $color-light-blue;
-
-    h3::before {
-      content: fa-content($fa-var-circle);
-      color: $color-light-blue;
-    }
-  }
-
-  &-update section.status {
-    border-color: $color-gold-light;
-
-    h3::before {
-      content: fa-content($fa-var-exclamation-circle);
-      color: $color-gold-light;
-    }
-  }
+  @include make-status-styles(
+    $status: "closed",
+    $color: $color-black,
+  );
+
+  @include make-status-styles(
+    $status: "decision",
+    $color: $color-red,
+  );
+
+  @include make-status-styles(
+    $status: "enrolled",
+    $color: $color-green,
+    $icon: $fa-var-check,
+    $top: 1px,
+    $left: 2.5px,
+    $size: 14px,
+  );
```

Comment:
Ah very cool mixin!

Reply:
more or less stole it from the existing `make-hub` mixin :)